.github/release-please/config.json

```diff
@@ -3,15 +3,17 @@
     "prerelease": false,
     "include-component-in-tag": false,
     "changelog-sections": [
-        { "type": "feat", "section": "Features", "hidden": false },
-        { "type": "fix", "section": "Bug Fixes", "hidden": false },
-        { "type": "docs", "section": "Documentation", "hidden": false },
-        { "type": "build", "section": "Build Related", "hidden": false },
-        { "type": "chore", "section": "Chores", "hidden": false },
-        { "type": "perf", "section": "Chores", "hidden": false },
-        { "type": "ci", "section": "Chores", "hidden": false },
-        { "type": "refactor", "section": "Chores", "hidden": false },
-        { "type": "test", "section": "Chores", "hidden": false }
+        { "type": "feat", "section": "🌟 Features", "hidden": false },
+        { "type": "fix", "section": "🩹 Fixes", "hidden": false },
+        { "type": "docs", "section": "📚 Documentation", "hidden": false },
+        
+        { "type": "chore", "section": "🧹 Chores", "hidden": false },
+        { "type": "perf", "section": "🧹 Chores", "hidden": false },
+        { "type": "refactor", "section": "🧹 Chores", "hidden": false },
+        { "type": "test", "section": "🧹 Chores", "hidden": false },
+
+        { "type": "build", "section": "🤖 Automation", "hidden": false },
+        { "type": "ci", "section": "🤖 Automation", "hidden": true }
```

Comment:
I also changed this to hidden as the CI changes do not affect the package output :thinking: